init.c

```diff
@@ -785,6 +830,8 @@ LUA_EXTERNC DLL_EXPORT int luaopen_libcutorch(lua_State *L);
 int luaopen_libcutorch(lua_State *L)
 {
   lua_newtable(L);
+  lua_pushvalue(L, -1);
+  lua_setglobal(L, "cutorch");
```

Comment:
Why are you making this global? init.lua already creates a global "cutorch" table. And in fact we want to avoid creating globals altogether (see [Torch Roadmap](https://github.com/torch/torch7/blob/master/ROADMAP.md#versioning--packaging))